README.md

```diff
@@ -24,6 +24,19 @@ yo phovea
  * Yeoman can be too opinionated at times but is easily convinced not to be.
  * Feel free to [learn more about Yeoman](http://yeoman.io/).
 
+## Commands
+
+| Command                    | Description  |
+|----------------------------|--------------|
+| `yo phovea` or `yo phovea:app` | Initialize a new web application, library bundle, web library, server library, or service |
+| `yo phovea:add-dependency` | Adds an additional Phovea plugins or external libraries to the current Phovea plugins |
+| `yo phovea:add-extension`  | Adds an additional web or server extension to the current Phovea plugins |
+| `yo phovea:clone`          | Clones a given Git repository and resolves the dependencies |
```

Comment:
the argument is not a git repository but the name of a phovea plugin that should be cloned